### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by andrewpmk**
Update translations

### CODE REVIEW

File: src/locales/fa/translation.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+    "12plusyearolds": "لطفا این ردیف را در گوگل ترنسلیت (به عنوان یک متن انگلیسی) کپی کنید. پس از آن لطفا نتیجه حاصل را در اینجا بازنشانی (پیست) کنید و سپس تصحیح نمایید.",
+    "5to11yearolds": "لطفا این ردیف را در گوگل ترنسلیت (به عنوان یک متن انگلیسی) کپی کنید. پس از آن لطفا نتیجه حاصل را در اینجا بازنشانی (پیست) کنید و سپس تصحیح نمایید.",
+    "address": "لطفا این ردیف را در گوگل ترنسلیت (به عنوان یک متن انگلیسی) کپی کنید. پس از آن لطفا نتیجه حاصل را در اینجا بازنشانی (پیست) کنید و سپس تصحیح نمایید.",
```
Comment (ericherscovich) [inline src/locales/fa/translation.json]: These werent translated correctly

File: src/locales/he/translation.json [hunk 2]
```diff
@@ -1,7 +1,10 @@
 {
+    "12plusyearolds": "12 + בן שנה",
+    "5to11yearolds": "5-11 שנה",
```
Comment (ericherscovich) [inline src/locales/he/translation.json:3]: Hebrew is right to left as well but I think it should be fine having it like this to be honest

File: src/locales/fa/translation.json (hunk 1 above)
Comment (andrewpmk) [reply src/locales/fa/translation.json]: fixed

File: src/locales/he/translation.json (hunk 2 above)
Comment (andrewpmk) [reply src/locales/he/translation.json:3]: this is Google translate, I don't speak Hebrew

Comment (ericherscovich) [reply src/locales/he/translation.json:3]: Ya should be fine, message gets across well enough

**Review by ericherscovich — APPROVED** [review]

**Pull request merged**